Lesson Plans/Private/XC.9 Emergency Operations.md

```diff
@@ -23,42 +23,107 @@ Develop knowledge of all abnormal and emergency procedures for the training airc
 > Can be completed at the same time as [[XC.8 Emergency Flight by Instruments - Four Fundamentals]] or [[XC.10 Lost Procedures and Diversion to Alternates]].
 
 ### Lesson Elements
+##### Handling Emergencies
 1. [[FAR 91.3 PIC Responsibility and Authority]]
-3. What is an emergency? When should you declare one?
+2. What is an emergency? When should you declare one?
 	1. When the outcome of the flight is in doubt
-4. Instrument failures (vacuum or pitot-static system)
-5. Fire
-	1. Engine fire
-		1. In air - Emergency Descent (this is the rapid descent procedure)
-			1. Start descent (bank 30°-45°; airspeed >=100 kts; should have positive load factor)
-			2. Then once fire out, execute forced landing (see [[SOLO.15 Emergency Landings]])
-		2. During start on ground
-	2. Electrical Fire
-	3. Wing Tip Fire
-6. Failure
-	1. Engine
-		1. During flight: see [[SOLO.15 Emergency Landings]]
-		2. During takeoff roll
-		3. Immediately after takeoff: Land straight ahead!^[[Cub Theatre: The-Not-That-Impossible Turn](https://www.youtube.com/watch?v=byBh1Qd6IgA) from [[Paul Bertorelli]] of [[AVweb]]]
-	2. Electrical
-		1. Malfunctions or complete failure
-		2. Nordo (No Radio)
-	3. Minimum fuel
-	4. Flap failure
-	5. Retractable gear malfunction
-	6. Door opening in flight
-	7. Low oil pressure
-7. Unsafe Conditions
-	1. [[VFR]] into [[IMC]]
-	2. Icing
-		1. [[How and Why Carburetor Ice Forms]]
-8. Emergency equipment and survival gear
-9. Emergency squawk codes: [[Squawk 7700]], [[Squawk 7600]], [[Squawk 7500]]
+	2. When you need higher priority handling from ATC
+
+##### Something is Wrong!
+DISCLAIMER: The POH has procedures for many of these situations, and you should follow them.  This bulleted list is intended to help you think through the list of emergencies in a logical way, debug the problem, and understand at a high level how you'll handle the situation.  **(POH)** means the POH has a procedure you should learn, with initial steps (in bold) that you need to memorize to execute quickly.
+
+- On the ground...
+	- Fire! - **(POH)** Cut off fuel to the engine; grab fire extinguisher; evacuate
+- During takeoff...
+	- Fire!
+		- **(POH)** Can you abort takeoff?  Abort takeoff.
+		- Else, take off, and land straight out
+	- Engine Failure!
+		- **(POH)** Abort takeoff, or forced landing straight ahead.^[[Cub Theatre: The-Not-That-Impossible Turn](https://www.youtube.com/watch?v=byBh1Qd6IgA) from [[Paul Bertorelli]] of [[AVweb]]]
+	- Airspeed isn't "live"
+		- Abort takeoff, or fly the pattern and land normally
+	- A door opens
+		- Takeoff normally.  Deal with it in the air, or land normally
+- In the air...
+	- Fire!  Where?
```

Comment:
I think there's a way ot use the smell and color of smoke to figure out what it is, but can't confirm specifics off hand. Maybe worth adding this info though if you can find what FAA says